reflector.cpp

```diff
@@ -443,5 +446,8 @@ void CReflector::WriteXmlFile(std::ofstream &xmlFile)
 	}
 	// unlock
 	ReleaseUsers();
-	xmlFile << "</" << m_Callsign << " heard users>" << std::endl;
+	xmlFile << "</STATIONS>" << std::endl;
+	
+	// reflector end
+	xmlFile << "</REFLECTOR>" << std:endl;
```

Comment:
you forgot a colon (should be double) .. std::endl;

Reply:
Patched, thanks Dave!